pkgs/dkg/drand.go

```diff
@@ -101,9 +105,10 @@ type LocalOwner struct {
 	RSAPub      *rsa.PublicKey
 	Owner       common.Address
 	Nonce       uint64
-	done        chan struct{}
+	Done        chan struct{}
```

Comment:
I don't get why you made all the these members public? unless they are needed for external usage lets make them private again.